engine/src/test/java/org/camunda/bpm/engine/test/api/repository/ProcessDefinitionQueryTest.java

```diff
@@ -108,6 +111,36 @@ public void testQueryByInvalidDeploymentId() {
     repositoryService.createProcessDefinitionQuery().deploymentId(null);
   }
 
+  @Test
+  public void testQueryByDeploymentTimeAfter() {
```

Comment:
This test is not robust. It assumes that time elapses between the deployments, but that is not enforced. Furthermore, it does not test that the number of returned definitions is in any way correct, e.g. if the process definition query returned no or too little results, the test would still succeed.